apis/proto/v1/vald/flush.proto

```
@@ -26,9 +26,19 @@ option java_multiple_files = true;
 option java_outer_classname = "ValdFlush";
 option java_package = "org.vdaas.vald.api.v1.vald";
 
-// Flush service provides ways to flush all indexed vectors.
+// Overview
+// Flush Service is responsible for removing all vectors that are indexed and uncommitted in the `vald-agent`.
 service Flush {
```

buf (1.47.2)

Service name "Flush" should be suffixed with "Service".

(SERVICE_SUFFIX)